src/index.js

```diff
@@ -1,13 +1,13 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import App from './App';
-
-
-const root = ReactDOM.createRoot(document.getElementById('root'));
-root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
-);
-
-
+import React from 'react';

```

Comment:
You don't change this file often, 
This is where the React world meets the traditional HTML/JS world